packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -47,26 +59,34 @@ const defaultProps = {
   textEmphasis: 'default',
   isTruncated: false,
   accessoryAlignment: 'alignCenter',
+  accessoryScale: '1em',
```

Comment:
If we use em as the default unit when the max is in px, is there a chance for large font sizes that this could be greater than the max scale?

Reply:
Does this last example answer your question?